ntcore/src/main/native/include/ntcore_c.h

```diff
@@ -1965,5 +1965,3 @@ void NT_Meta_FreeClients(struct NT_Meta_Client* arr, size_t count);
 #ifdef __cplusplus
 }  // extern "C"
 #endif
-
-#include "ntcore_c_types.h"
```

Comment:
Why are you removing this?

Reply:
It was not found when building for a wrapper